src/main/java/liquibase/ext/neo4j/database/Neo4jDatabase.java

```diff
@@ -137,22 +142,30 @@ public String getNeo4jVersion() {
         return String.valueOf(get(NEO4J_VERSION_ATTRIBUTE));
     }
 
-    private void setNeo4jVersion(String neo4jVersion) {
-        set(NEO4J_VERSION_ATTRIBUTE, neo4jVersion);
+    public boolean isEnterprise() {
+        return get(NEO4J_EDITION_ATTRIBUTE.toLowerCase(Locale.ENGLISH)).equals("enterprise");
     }
 
-    private void setCatalogSupport(String neo4jVersion) {
+    private void initializeServerAttributes() {
+        Map<String, ?> components = readComponents();
+        String neo4jVersion = (String) components.get("VERSION");
+        set(NEO4J_VERSION_ATTRIBUTE, neo4jVersion);
```

Comment:
please let me know if I'm abusing `liquibase.database.AbstractJdbcDatabase#attributes` in any way, @nvoxland